changes/4787.removed

```diff
@@ -0,0 +1 @@
+Removed support for `HIDE_RESTRICTED_UI` config.
```

Comment:
Would be good to elaborate here - specifically make it clear whether the effect is 1) restricted UI elements are never hidden or 2) restricted UI elements are always hidden. (I believe it's the latter now?)